client/src/file/mod.rs

```diff
@@ -118,15 +160,35 @@ impl Keyring {
             }
         };
 
-        let key = keyring.derive_key(&secret)?;
-        for encrypted_item in &mut keyring.items {
-            if let Err(_err) = encrypted_item.clone().decrypt(&key) {
+        if validate_items {
+            let key = keyring.derive_key(&secret)?;
+
+            let mut n_broken_items = 0;
+            let mut n_valid_items = 0;
+            for encrypted_item in &keyring.items {
+                if let Err(_err) = encrypted_item.clone().decrypt(&key) {
+                    #[cfg(feature = "tracing")]
+                    tracing::error!("Failed to decrypt item, incorrect secret?: {_err}");
```

Comment:
I was talking about putting this one as `warn`.

Reply:
Indeed. Fixed